### PR TITLE
Circuit Playground auto bike brake light

### PR DESCRIPTION
**Pull request by dherrada**
Works best with CPB because it's faster, but it still works fine on the CPX

### CODE REVIEW

File: Circuit_Playground_Brake_Light/code.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            # Detects shake. Due to the very low shake threshold, this alone would have very low
+            # specificity. This was mitigated by having it only run when the acceleration
+            # difference is greater than 1 m/s^2 at least 3 times in a row.
+            if not cp.shake(shake_threshold=10):
+                cp.pixels.brightness = 1
+                start = time.monotonic()
+                sleep = True
+        i += 1
+
+    # sleep variable is for short circuiting. Not really necessary, but it makes things run faster
```
Comment (kattni) [inline Circuit_Playground_Brake_Light/code.py:44]: Using `sleep` as the variable name could be confused with `time.sleep()`.

File: Circuit_Playground_Brake_Light/code.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import time
+import math
+from adafruit_circuitplayground import cp
+
+brightness = 0
+l10 = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```
Comment (siddacious) [inline Circuit_Playground_Brake_Light/code.py:6]: I'd change these to more meaningful names and have a comment about their use

File: Circuit_Playground_Brake_Light/code.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+i = 0
+cp.pixels.fill((255, 0, 0))
+
+light_on = False
+
+while True:
+    x, y, z = cp.acceleration
+
+    # moving average n=10, not super smooth, but it substantially lowers the amount of noise
```
Comment (siddacious) [inline Circuit_Playground_Brake_Light/code.py:21]: The rest of the code could use more explanation of what is going on and why. If you are going to go though each section of the code in the guide,  this is ok as is (though comments with more explanation never hurt)

Comment (dherrada) [reply Circuit_Playground_Brake_Light/code.py:21]: My plan was to explain it in the guide, but I'll add some more explanations

**Pull request merged**